src/commands/user/lookup.ts

```diff
@@ -19,7 +19,7 @@ export async function lookUp(message: Message, args: string[]) {
 			break
 	}
 
-	const response = await axios.get(apiURL)
+	const response = await axios.get(apiURL, {validateStatus: () => true})
```

Comment:
Validate status is a bit ignored when it comes to status codes...

Here's the official example from Axios:
```js
  validateStatus: function (status) {
    return status >= 200 && status < 300; // default
  },

```

I suggest you change these in the next PR, just for a better validation.